tests/openvino/utils_tests.py

```diff
@@ -63,6 +63,7 @@
     "donut": "fxmarty/tiny-doc-qa-vision-encoder-decoder",
     "donut-swin": "hf-internal-testing/tiny-random-DonutSwinModel",
     "detr": "hf-internal-testing/tiny-random-DetrModel",
+    "ernie4_5": "snake7gun/tiny-random-ernie4_5",
```

Comment:
please update test files: `test_export.py`, `test_export_cli.py` and `test_modeling.py`

Reply:
I just wonder why 'test_export.py, test_export_cli.py' are needed in this case, looks they are only designed for task oriented test ?

Reply:
done